### PR TITLE
Example aedt

### PR DESCRIPTION
**Pull request by pvinod13**
Hi , 

I have added multiphysics-example involving AEDT. Please have a look and let me know if it is ok..!

### CODE REVIEW

File: examples/ansys-aedt-workflow/main.py
Comment (franknli) [inline examples/ansys-aedt-workflow/main.py:24]: Please either remove the "server_workdir" argument (it is optional anyway and has its default value if unspecified), or change its value to be different from that of "client_workdir".  @pvinod13

File: examples/ansys-aedt-workflow/main.py
Comment (franknli) [inline examples/ansys-aedt-workflow/main.py:44]: Remove the commented-out line!

File: examples/ansys-aedt-workflow/main.py
Comment (franknli) [inline examples/ansys-aedt-workflow/main.py:33]: No need to upload this file: the only time it is used is to be the argument of run_script_file call.

File: examples/ansys-aedt-workflow/assets/pulse_data.wbjn
Comment (franknli) [inline examples/ansys-aedt-workflow/assets/pulse_data.wbjn:1]: Never use the absolute path: how would you expect other people to run this example if it refers to absolute path? @pvinod13

File: examples/ansys-aedt-workflow/main.py
Comment (franknli) [inline examples/ansys-aedt-workflow/main.py:48]: This is not how file location should be defined in WB scripts. To reference a data file in WB script:
(1) upload the data file to PyWB server. And the file will be uploaded to the server's working directory.
(2) in your WB script, define the file location this way:
```
import os
work_dir = GetServerWorkingDirectory()
data_file_path = os.path.join(work_dir, the_file_name)
```

File: examples/ansys-aedt-workflow/main.py
Comment (franknli) [inline examples/ansys-aedt-workflow/main.py:19]: Line 18 and 19 can be removed - the variables are not used anyway.

**Review by franknli — CHANGES_REQUESTED** [review]

**Comment by pvinod13** [comment]
Modified the scripts as per the suggestion in the commit (bca51b6c43c5492787c16fd070b13ce282430c9b)

File: examples/ansys-aedt-workflow/scripts/DC_Cond_ThermTransient_VariableTimeStep.py
Comment (franknli) [inline examples/ansys-aedt-workflow/scripts/DC_Cond_ThermTransient_VariableTimeStep.py:232]: You should not hard-code any file path (line 231), or use a file path defined on the client side (line 232). So please replace line 231 and 232 with correct way to accessing data file on server:
(1) Upload the data file (the csv in this case) from the client to the server - you have already done this in main.py.
(2) In the script run by the server (like this script), you assume that the file is in the server's working directory:
```
work_dir = GetServerWorkingDirectory()
csv_file = os.path.join(work_dir, "10_100_Pulse.csv") 
input_values = csv.reader(open(csv_file))
```

File: examples/ansys-aedt-workflow/main.py
Comment (franknli) [inline examples/ansys-aedt-workflow/main.py:44]: Line 36 - 44 is NOT needed at all. See my explanation below in the other script file. Basically, client side code (like this one) should NOT define absolute file path used by server side code - client and server can run on two different machines!

File: examples/ansys-aedt-workflow/main.py
Comment (franknli) [inline examples/ansys-aedt-workflow/main.py:22]: There is usually NO NEED to specify server_workdir, rather just let server uses the default work dir on the server computer. Furthermore, server working directory should NEVER be the same as the client - otherwise there is no need to upload and download files!

You can simply remove the server_workdir augument.

**Review by franknli — CHANGES_REQUESTED** [review]

File: examples/ansys-aedt-workflow/scripts/DC_Cond_ThermTransient_VariableTimeStep.py
Comment (pvinod13) [reply examples/ansys-aedt-workflow/scripts/DC_Cond_ThermTransient_VariableTimeStep.py]: modified as per the suggestion

File: examples/ansys-aedt-workflow/main.py
Comment (pvinod13) [reply examples/ansys-aedt-workflow/main.py]: modified as per the suggestion

File: examples/ansys-aedt-workflow/main.py
Comment (pvinod13) [reply examples/ansys-aedt-workflow/main.py]: modified as per the suggestion

**Comment by pvinod13** [comment]
Modified the scripts as per the suggestion in the commit (https://github.com/ansys/pyworkbench-examples/pull/52/commits/3bacfb976c15e1ea5c4251c34be98188cf204ccd). Can you revisit the PR..!

**Review by franknli — APPROVED** [review]
Looks good. Thanks for following up with the suggestions

**Comment by franknli** [comment]
@pvinod13  There seems to be some issue with the doc build. You can check with @Revathyvenugopal162 and see if any fix is needed.

**Comment by Revathyvenugopal162** [comment]
@pvinod13 The workflow is fixed, as i told feel free to add a thumbnail image, if you wish.

**Review by Revathyvenugopal162 — APPROVED** [review]

**Review by franknli — APPROVED** [review]

**Comment by Revathyvenugopal162** [comment]
@franknli @pvinod13 can we merge this PR, from my side it is ready to merge. we can merge this once you confirm this.

**Comment by franknli** [comment]
> @franknli @pvinod13 can we merge this PR, from my side it is ready to merge. we can merge this once you confirm this.

It is OK from my side - which is why I already approved it. It seems to be requiring additional approvals. @PipKat

**Pull request merged**
